operator/internal/pkg/custom/verify.go

```diff
@@ -39,7 +39,7 @@ func (r *Resource) CheckFn(ctx context.Context, manifestObj *unstructured.Unstru
 	}
 
 	ready, err := customStatus.WaitForCustomResources(ctx, resource)
-	if err != nil {
+	if client.IgnoreNotFound(err) != nil {
```

Comment:
is there a reason for doing this?

Reply:
oh, it's changes in my load test setup, should not be here, I reverted it. thanks for pointing out.